conda-envs/environment-dev-py38.yml

```diff
@@ -22,8 +22,7 @@ dependencies:
 - pytest-cov>=2.5
 - pytest>=3.0
 - python-graphviz
-- python=3.8
```

Comment:
Why was the python version dependency removed?

Reply:
Yeah that's not right.